src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js

```diff
@@ -31,10 +31,10 @@ export type Props = {
 
 export const getFetchStatus = statusFetchFactory(types.FETCH_PULL_REQUEST_DISCUSSION)
 
-export const getLoggedUser = (state: Object) => state.session.profile.fullName
+export const getLoggedUsername = (state: Object) => state.session.profile.fullName
```

Comment:
There's another, different `getLoggedUsername` in `ducks/session/selectors`:
`export const getLoggedUsername = state => _.get(state, ['entities', 'me', 'username'], null)`
If they're equivalent, maybe we should replace the one we have here with the one from `ducks/session/selectors`.